lib/tasks/sync.rake

```diff
@@ -10,11 +10,15 @@ namespace :sync do
     theme_cache = Theme.all.pluck(:title, :id).to_h
     count = 0
 
+    puts "#{Time.now} - START: Attempting to sync legacy datasets"
+
     get_packages "https://data.gov.uk" do |package|
       MetadataTools.add_dataset_metadata(package, orgs_cache, theme_cache)
-      print "Imported #{count+=1} datasets...\r"
+      puts "Imported #{count+=1} datasets...\r"
     end
 
+    puts "#{Time.now} - SUCCESS: Finished syncing #{count} legacy datasets"
```

Comment:
maybe worth writing this to the logs?

```
  logger = Logger.new
  logger.info("#{Time.now} - START: Attempting to sync legacy 
  datasets")

  # rake task runs ...

  logger.info("#{Time.now} - SUCCESS: Finished syncing #{count} 
  legacy datasets")
```

Reply:
also a good idea to log any errors that we might get during the sync rake task.

Reply:
Cool